pages/deposit/[symbol].tsx

```diff
@@ -180,13 +172,17 @@ const Deposit: NextPage<
                     Your interest payout will be
                   </label>
                   <CountUp
-                    end={
-                      +formatUnits(expectedProceeds, asset.underlying.decimals)
-                    }
+                    end={Number(expectedProceeds)}
                     suffix={` ${asset.underlying.symbol}`}
                     className={`flex gap-2 ${
                       epochs === 0 ? 'text-gray-400' : ''
                     } text-xl`}
+                    formattingFn={(value) =>
+                      `${formatUnits(
+                        BigInt(value),
+                        asset.underlying.decimals,
+                      )} ${asset.underlying.symbol}`
+                    }
```

Comment:
@onemonster Could you please review this?: The number is always reset when the user permit or signs the transaction.

Reply:
This is because the user balance is 0 and value is set to 0 since it's min(value, user balance) is used

Reply:
https://github.com/clober-dex/coupon.finance/assets/10525011/8b92d6cc-d461-48d9-9f4c-7a0ccd7de608

Nope please check this video